python-threatexchange/threatexchange/extensions/video_vpdq/vpdq_brute_hasher.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import vpdq
+import typing as t
+from .vpdq_util import dedupe, quality_filter, VPDQMatchResult
+
+BITS_IN_VPDQ = 256
+
+
+def match_VPDQ_in_another(
+    hash1: t.List[vpdq.VpdqFeature],
+    hash2: t.List[vpdq.VpdqFeature],
+    distance_tolerance: int,
+) -> int:
+    """Count matches of hash1 in hash2
+
+    Args:
+        hash1 (list of VPDQ feature)
+        hash2 (list of VPDQ feature)
+        distance_tolerance (int):The hamming distance tolerance of between two frames.
+        If the hamming distance is bigger than the tolerance, it will be considered as unmatched
+
+    Returns:
+        int: The count of matches of hash1 in hash2
+    """
+    return sum(
+        any(h1.hamming_distance(h2) <= distance_tolerance for h2 in hash2)
+        for h1 in hash1
+    )
+
+
+def match_VPDQ_hash_brute(
+    target_hash: t.List[vpdq.VpdqFeature],
+    query_hash: t.List[vpdq.VpdqFeature],
```

Comment:
(Nit/ignorable): I'm not sure target/query is a common term. I've seen: 
* Needle / haystack (string functions esp)
* matched/banked for MMS (doesn't apply here because we don't have bank)

There's probably others. Is the target/query common in other usecases?

Reply:
In vpdq/readme, the two terms used are query and comparison(target). I will align with the readme.